config/settings/base.py

```diff
@@ -41,7 +41,7 @@
 # ------------------------------------------------------------------------------
 # https://docs.djangoproject.com/en/dev/ref/settings/#databases
 
-DATABASES = {"default": env.db("DATABASE_URL", default="postgres:///care")}
+DATABASES = {"default": env.db("DATABASE_URL", default="postgis:///care")}
```

Comment:
Do we really need this, since postgis is an extension on top of postgres

Reply:
I checked. This seems to be the way. The standard looks like the below
```
DATABASES = {
    'default': {
        'ENGINE': 'django.contrib.gis.db.backends.postgis',
        'NAME': 'YOUR_POSTGIS_DATABASE_NAME',
        'USER': 'YOUR_DATABASE_USER_NAME',
        'PASSWORD': 'YOUR_DATABASE_USER_PASSWORD',
        'HOST': 'localhost',
        'PORT': '5432'
    }
}
```

Since we're using url directly, what I've done seems to be the way.

Reply:
You might have to change the value of `DATABASE_URL` in heroku app env to do the same.